include/pgduckdb/pgduckdb_types.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "duckdb/common/types/data_chunk.hpp"
```

Comment:
Without this fix, all .cpp files must include `duckdb/common/types/data_chunk.hpp` (or any file that includes it) before including `pgduckdb/pgduckdb_types.hpp`
This is undesirable, and the header file should be self-contained

Reply:
Fair, but in this case we should clean the cpp files where `pgduckdb_types.hpp` is included to remove the extra duckdb includes no?

Reply:
We don't include `data_chunk.hpp` anywhere else. That brings two questions:
1. Should this header simply include `duckdb.hpp` instead of a specific header?
2. @Y-- your suggestion for removal then becomes: should we remove `duckdb.hpp` from `cpp` files that already include `pgduckdb_types.hpp`?

Reply:
Right I think if we're going to make this header self-contained. Or maybe better we should forward-declare the various DuckDB classes we're using in the functions below?

Reply:
Forward declaration is quite a hassle, and I don't really see the benefit of it for duckdb classes (as opposed to postgres structs).

So, then what's your answer to those two questions?
1. Yes
2. Yes

Because that would be fine by me.

Reply:
Yes and yes :-)

Reply:
Forgot to submit my comments..
Could you check my previous comments?
If you feel strongly to include entire `duckdb.hpp` instead of specific headers in `pg_duckdb`, I can proceed with the proposed changes

Reply:
I'm fine with the specific headers (and definitely agree with your point, it's much better)